rust/lance-encoding/src/encodings/logical/primitive.rs

```diff
@@ -576,6 +612,17 @@ impl StructuralPageDecoder for SimpleAllNullPageDecoder {
     }
 }
 
+#[derive(Debug, Clone)]
+struct MiniBlockSchedulerDictionary {
+    // These come from the protobuf
+    dictionary_decompressor: Arc<dyn BlockDecompressor>,
+    dictionary_buf_position_and_size: (u64, u64),
+    dictionary_data_alignment: u64,
```

Comment:
What is this alignment for?

Reply:
this `dictionary_data_alignment` is used later for converting the raw bytes::Bytes fetched from disk IO to LanceBuffer
```
                        LanceBuffer::from_bytes(
                            dictionary_data,
                            dictionary.dictionary_data_alignment,
                        ),
```